lib/geo_combine/cli.rb

```diff
@@ -0,0 +1,23 @@
+require 'thor'
+require 'rake'
+
+Dir.glob('lib/tasks/geo_combine.rake').each { |r| load r }
+
+module GeoCombine
+  class CLI < Thor
+    desc 'clone', 'Clone all OpenGeoMetadata repositories'
+    def clone
+      Rake::Task['geocombine:clone'].invoke
+    end
+
+    desc 'pull', '"git pull" OpenGeoMetadata repositories'
+    def pull
+      Rake::Task['geocombine:pull']
```

Comment:
Yeah, there also appears to be a problem with loading the rake tasks if you're not in the gem's directory. Fixes incoming.